src/frontend/screens/AppPasscode/EnterPassToTurnOff.tsx

```diff
@@ -1,16 +1,16 @@
 import * as React from "react";
-import { defineMessage } from "react-intl";
+import { defineMessages } from "react-intl";
```

Comment:
need to do the same thing for `LeaveProjectInitial.tsx`

Reply:
OH, yes, its on my local device, I thought i pushed it with this

Reply:
https://github.com/digidem/mapeo-mobile/pull/1053/commits/c8f9fc44cdfbf9ad9b949126a06a11cac8b5c8cf